doc/api/async_hooks.md

```diff
@@ -192,19 +192,8 @@ Every new resource is assigned a unique ID.
 ###### `type`
 
 The `type` is a string that represents the type of resource that caused
-`init` to call. Generally it will be the name of the resource's constructor.
-The resource types provided by the built-in Node.js modules are:
-
-```
-FSEVENTWRAP, FSREQWRAP, GETADDRINFOREQWRAP, GETNAMEINFOREQWRAP, HTTPPARSER,
-JSSTREAM, PIPECONNECTWRAP, PIPEWRAP, PROCESSWRAP, QUERYWRAP, SHUTDOWNWRAP,
-SIGNALWRAP, STATWATCHER, TCPCONNECTWRAP, TCPWRAP, TIMERWRAP, TTYWRAP,
-UDPSENDWRAP, UDPWRAP, WRITEWRAP, ZLIB, SSLCONNECTION, PBKDF2REQUEST,
-RANDOMBYTESREQUEST, TLSWRAP
-```
-
-There is also the `PROMISE` resource type, which is used to track `Promise`
-instances and asynchronous work scheduled by them.
+`init` to call. Generally it will correspond the name of the resource's
```

Comment:
I know it was there before but probably should `to call` -> `to be called`
Less sure but maybe `correspond` -> `correlate`
Correspond: have a close similarity; match or agree almost exactly.
Correlate: have a mutual relationship or connection, in which one thing affects or depends on another.